docs/sources/tutorials/send-metrics-to-prometheus.md

```diff
@@ -0,0 +1,182 @@
+title: Use Grafana Alloy to send metrics to Prometheus
+menuTitle: Send metrics to Prometheus
+weight: 15
+---
+# Use Grafana Alloy to send metrics to Prometheus
+In the [Get started with {{< param "PRODUCT_NAME" >}} tutorial][get started], you learned how to configure {{< param "PRODUCT_NAME" >}} to collect and process logs from your local machine and send them to Loki, running in the local Grafana stack. 
+
+As a next step, you will collect and process metrics from the same machine using {{< param "PRODUCT_NAME" >}} and send them to Prometheus, running in the same Grafana stack. 
+
+This process will enable you to query and visualize the metrics sent to Prometheus using the Grafana dashboard.
+
+## Prerequisites
+
+Complete the [previous tutorial][get started] to:
+1. Install {{< param "PRODUCT_NAME" >}} and start the service in your environment.
+1. Set up a local Grafana instance.
+1. Create a `config.alloy` file.
+
+## Configure {{% param "PRODUCT_NAME" %}}
+
+Once the prerequisite steps have been completed, the next step is to configure {{< param "PRODUCT_NAME" >}}.
+
+Same as you did for logs, you will use the components in the `config.alloy` file to tell {{< param "PRODUCT_NAME" >}} which metrics you want to scrape, how you want to process that data, and where you want the data sent.
+
+Add the following to the `config.alloy` file you created in the prerequisite steps.  
+
+### First component: Scraping
+
+Paste this component into the top of the `config.alloy` file:
+
+```alloy
+prometheus.exporter.unix "local_system" { }
+
+prometheus.scrape "scrape_metrics" {
+  targets    = prometheus.exporter.unix.local_system.targets
+  forward_to = [prometheus.relabel.filter_metrics.receiver]
+  scrape_interval = "10s"
+}
+```
+This configuration defines a Prometheus exporter for a local system from which the metrics will be collected. 
+
+It also creates a [`prometheus.scrape`][prometheus.scrape] component named `scrape_metrics` which does the following:
+
+1. It connects to the `local_system` component (its "source" or target).
+1. It forwards the metrics it scrapes to the "receiver" of another component called `filter_metrics` which you will define next.
+1. It tells {{< param "PRODUCT_NAME" >}} to scrape metrics every 10 seconds. 
+
+### Second component: Filter metrics
+
+Filtering non-essential metrics before sending them to a data source can help you reduce costs and enable you to focus on the data that matters most. The filtering strategy of each organization will differ as they have different monitoring needs and setups. 
+
+The following example demonstrates filtering out or dropping metrics before sending them to Prometheus. 
+
+Paste this component next in your configuration file:
+```alloy
+prometheus.relabel "filter_metrics" {
+   rule {
+    action = "drop"
+    source_labels =[ "env"]
+    regex = "dev"
+  }
+ forward_to = [prometheus.remote_write.metrics_service.receiver]
+}
+```
```

Comment:
Adjusted to align with what `alloy fmt` would produce:

```suggestion
```alloy
prometheus.relabel "filter_metrics" {
  rule {
    action        = "drop"
    source_labels = ["env"]
    regex         = "dev"
  }
  
  forward_to = [prometheus.remote_write.metrics_service.receiver]
}
```
```